README.rst

```diff
@@ -81,6 +82,8 @@ To enable the `Cybersource <https://cybersource.com>`__ payment processor, two k
 
 The ``merchant_id`` setting corresponds to your Merchant ID.
 
+If you are running in production, you need to change the ``flex-run-environment`` value in the configuration file as well. You should change the ``flex-run-environment`` value to ``cybersource.environment.production``
```

Comment:
please change the name here also.
`flex-run-environment` -> `flex_run_environment`